app/grandchallenge/core/context_processors.py

```diff
@@ -41,6 +46,10 @@ def challenge(request):
         )
         .with_overdue_status()
         .status_aggregates,
+        "num_outstanding_invoices": Invoice.objects.filter(challenge=challenge)
+        .exclude(payment_type=PaymentTypeChoices.COMPLIMENTARY)
+        .filter(payment_status=PaymentStatusChoices.ISSUED)
```

Comment:
Shouldn't we include all but paid invoices here?

Reply:
Ah no, I think this is correct. It doesn't make sense to include initialized and requested here since those haven't been sent to the challenge orgs yet of course. Sorry!